extensions/wikia/CreateNewWiki/FinishCreateWikiController.class.php

```diff
@@ -144,11 +101,11 @@ private function getClassicMainPage( $mainArticle ) {
 		global $wgParser, $wgSitename;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Do not use the global wgParser Parser instance. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AWgParserUsage)